src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -432,7 +432,7 @@ def update_ag_backend_http_settings_collection(instance, parent, item_name, port
         instance.request_timeout = timeout
     if connection_draining_timeout is not None:
         instance.connection_draining.enabled = bool(connection_draining_timeout)
-        instance.connection_draining.drain_timeout_in_sec = connection_draining_timeout
+        instance.connection_draining.drain_timeout_in_sec = connection_draining_timeout or 1
```

Comment:
Do we have to set `1` when users used `0` to disable the connection_draining? A bit confusing, that you can use `1` to enable; and after you use `0` to disable, but the next read still shows `1` like things not cleaned up. A service side glitch?